.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
This is specific to your setup so I think it should be mentioned in your `core.excludesFile` variable in your `~/.gitconfig` file. We can't list here all combinations of build directory names :)